*: update "change column" and "begin" (#2922)
pingcap/docs: sql-statements/sql-statement-begin.md

## MySQL compatibility

> TiDB can carry the option of `PESSIMISTIC` or `OPTIMISTIC` when executing the `BEGIN` statement to indicate the type of transaction this statement opens.

Reviewer: ```suggestion
You can add the `PESSIMISTIC` or `OPTIMISTIC` option to the `BEGIN` statement in TiDB to indicate the type of transaction that this statement starts.
```
